edx_name_affirmation/tasks.py

```diff
@@ -17,14 +17,18 @@
 
 log = logging.getLogger(__name__)
 
+DEFAULT_RETRY_DELAY = 30
```

Comment:
naming nit - without reading the code for celery we can't tell how long this is. DEFAULT_RETRY_SECONDS or similar helps future readers